config/samples/certification_v1alpha1_operatorpipeline.yaml

```diff
@@ -5,4 +5,6 @@ metadata:
 spec:
   openShiftPipelineVersion: v1.5.0
   operatorPipelinesRelease: main
+  KubeconfigSecretName: "kubeconfig"
   gitHubSecretName: "github-api-token"
+  PyxisSecretName: "pyxis-api-secret"
```

Comment:
These should be camel case to match `CSV`.

Reply:
woops fixed.